docs/src/main/paradox/server.md

```diff
@@ -1,220 +0,0 @@
-
-Java
-:  @@snip [GreeterServiceImpl.java]($root$/../plugin-tester-java/src/main/java/example/myapp/helloworld/GreeterServiceImpl.java) { #full-service-impl }
-
-### Server
-
-That service can then be handled by an Akka HTTP server via the generated `GreeterServiceHandler`,
-which is a function from `HttpRequest` to @scala[`Future[HttpResponse]`]@java[`CompletionStage<HttpResponse>`].
-It returns 404 as response for requests that don't match the path of the service.
-
-The server will run one instance of the implementation and that is then shared between requests,
-this mean that it must be thread safe. In the sample above there is no mutable state, for more about safely implementing
-servers with state see [stateful](#stateful-services)
-
-A main program that starts a Akka HTTP server with the `GreeterService` looks like this:
-
-Scala
-:  @@snip [GreeterServiceImpl.scala]($root$/../plugin-tester-scala/src/main/scala/example/myapp/helloworld/GreeterServer.scala) { #full-server }
-
-Java
-:  @@snip [GreeterServiceImpl.java]($root$/../plugin-tester-java/src/main/java/example/myapp/helloworld/GreeterServer.java) { #full-server }
-
-Note that it's important to enable HTTP/2 in the configuration of the `ActorSystem`.
-
-```
-akka.http.server.preview.enable-http2 = on
-```
-
-# Multiple services
-
-When a server handles several services the handlers must be combined with
-@scala[`akka.grpc.scaladsl.ServiceHandler.concatOrNotFound`]@java[`akka.grpc.javadsl.ServiceHandler.concatOrNotFound`]
-
-Scala
-:  @@snip [GreeterServiceImpl.scala]($root$/../plugin-tester-scala/src/main/scala/example/myapp/CombinedServer.scala) { #concatOrNotFound }
-
-Java
-:  @@snip [GreeterServiceImpl.java]($root$/../plugin-tester-java/src/main/java/example/myapp/CombinedServer.java) { #import #concatOrNotFound }
-
-@scala[Note that the `GreeterServiceHandler.partial` and `EchoServiceHandler.partial` are used instead of `apply`
-methods to create partial functions that are combined by concatOrNotFound.]
-
-## Running
-
-To run the server with HTTP/2 enabled correctly, you will likely have to configure the Jetty ALPN
-agent as described @extref[in the Akka HTTP documentation](akka-http:server-side/http2.html#application-layer-protocol-negotiation-alpn-):
-
-See the detailed chapters on @ref[sbt](sbt.md#starting-your-akka-grpc-server-from-sbt), @ref[Gradle](gradle.md#starting-your-akka-grpc-server-from-gradle)
-and @ref[Maven](maven.md#starting-your-akka-grpc-server-from-maven) for details on adding the agent.
-
-
-## Stateful services
-
-More often than not, the whole point of the implementing a service is to keep state. Since the service implementation
-is shared between concurrent incoming requests any state must be thread safe.
-
-There are two recommended ways to deal with this:
-
- * Make the state immutable – immutable state that is created before or when the service is instantiated and then never changes is safe
```

Comment:
I removed that one, btw. Maybe I'm missing something but "stateful" to me implies that the state can change over time?

Instead, I pointed to CRUD-style programming where a DB handles concurrency.